grpc_codegen/avs_pb.d.ts

```diff
@@ -933,7 +932,7 @@ export namespace Task {
         memo: string,
         completedAt: number,
         maxExecution: number,
-        status: TaskStatus,
+        status: avs_pb.TaskStatus,
```

Comment:
I think this is some editor replace/all mistake. it should just be TaskStatus. becuase this is generated code.

Reply:
wow, good catch. I regenerated the protobuf js files and yes this is reverted.